packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -47,14 +48,18 @@ export class JaegerExporter implements SpanExporter {
         : 2000;
 
     // https://github.com/jaegertracing/jaeger-client-node#environment-variables
-    // By default, the client sends traces via UDP to the agent at localhost:6832. Use JAEGER_AGENT_HOST and
-    // JAEGER_AGENT_PORT to send UDP traces to a different host:port. If JAEGER_ENDPOINT is set, the client sends traces
-    // to the endpoint via HTTP, making the JAEGER_AGENT_HOST and JAEGER_AGENT_PORT unused. If JAEGER_ENDPOINT is secured,
-    // HTTP basic authentication can be performed by setting the JAEGER_USER and JAEGER_PASSWORD environment variables.
-    localConfig.endpoint = localConfig.endpoint || process.env.JAEGER_ENDPOINT;
-    localConfig.username = localConfig.username || process.env.JAEGER_USER;
-    localConfig.password = localConfig.password || process.env.JAEGER_PASSWORD;
-    localConfig.host = localConfig.host || process.env.JAEGER_AGENT_HOST;
+    // By default, the client sends traces via UDP to the agent at localhost:6832. Use OTEL_EXPORTER_JAEGER_AGENT_HOST and
+    // JAEGER_AGENT_PORT to send UDP traces to a different host:port. If OTEL_EXPORTER_JAEGER_ENDPOINT is set, the client sends traces
+    // to the endpoint via HTTP, making the OTEL_EXPORTER_JAEGER_AGENT_HOST and JAEGER_AGENT_PORT unused. If OTEL_EXPORTER_JAEGER_ENDPOINT is secured,
+    // HTTP basic authentication can be performed by setting the OTEL_EXPORTER_JAEGER_USER and OTEL_EXPORTER_JAEGER_PASSWORD environment variables.
+    localConfig.endpoint =
```

Comment:
please save first the value in to some variable - no need to parse it each time

Reply:
good call, will fix in the next batch of changes